packages/web-api/src/converters/scan-run-error-converter.ts

```diff
@@ -10,6 +10,16 @@ export class ScanRunErrorConverter {
             return ScanRunErrorCodes.urlNavigationTimeout;
         }
 
-        return ScanRunErrorCodes.internalError;
+        return this.createInternalErrorWithMessage(scanError);
+    }
+
+    private createInternalErrorWithMessage(appendMessage: string): ScanRunErrorCode {
```

Comment:
We do not want to send our internal error message to a client as is. We may need to create a new error code for the Non Success HTTP response and match our message to be translated to a client message. RegEx can take portion w/ HTTP code from a message. Ideally the HTTP response code should be part of separate property in error response type.